txprocessor/txprocessor.go

```diff
@@ -395,6 +395,10 @@ func (tp *TxProcessor) ProcessTxs(coordIdxs []common.Idx, l1usertxs, l1coordinat
 
 	// Process L2Txs
 	for i := 0; i < len(l2txs); i++ {
+		if l2txs[i].Type == common.TxTypeExit && l2txs[i].Amount.Cmp(big.NewInt(0)) == 0 {
```

Comment:
Is there any way to the amount be negative? Maybe we should filter this way:

```suggestion
		if l2txs[i].Type == common.TxTypeExit && l2txs[i].Amount.Cmp(big.NewInt(0)) <= 0 {
```

Reply:
Also, can we test this case?